### ISSUE TITLE
ValueError: `_table` isn't defined - the Table Metaclass should set it.

### ISSUE THREAD

**Issue by cheesycod** [piccolo-orm/piccolo#500]
class BotQueue(Table, tablename="bot_queue"):
    bot_id = BigInt(primary_key=True)
    username = Text(null=False)
    banner = Text(null=True)
    description = Text(null=False)
    long_description = Text(null=False)
    website = Text(null=True)
    invite = Text(null=True)
    added_at = Timestamptz(null=False, default=TimestamptzNow())
    state = Integer(choices=State, default=State.PENDING)
    list_source = ForeignKey(null=True, references=BotList, on_delete=OnDelete.cascade, on_update=OnUpdate.cascade)
    owner = BigInt(null=False)
    extra_owners = Array(base_column=BigInt(null=False))

**Comment by dantownsend** [comment]
@cheesycod I've been able to replicate this. Did you get the error when trying to run a migration?

Seems to be a problem with this:

```python
extra_owners = Array(base_column=BigInt(null=False))
```

**Comment by dantownsend** [comment]
This works:

```python
extra_owners = Array(base_column=Integer(null=False))
```

It's something to do with using `BigInt` - it must be failing because it's a subclass of `Integer`, and for some reason `_table` isn't being set properly.

**Comment by dantownsend** [comment]
Should be fixed now in the latest version o PyPI - 0.74.3.

Thanks for reporting the issue 👍
